sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -42,13 +42,15 @@ public static async Task Main(string[] args)
 
             // Instantiate the client
 
+            // These environment variables are necessary for application Id to login using a client secret.
+            Environment.SetEnvironmentVariable("AZURE_CLIENT_SECRET", options.ClientSecret);
```

Comment:
This only needs to be done for one of the client creations, so please move it to that method, but above the snippet region.

Also, do you think we should override any values that are there, if they already have a value set?

Reply:
I think override is better as we specifically ask for these in the options. It feels more common for people to expect those values to be used. Also, i'm adding it here as I thought we'll eventually have only one CreateClientMethod. I have not been able to get interactive mode to work and hence two methods exist now. Maybe it'll be more clear once I get both to work.